pkg/platform/aws/aws.go

```diff
@@ -410,3 +412,22 @@ func (c *config) checkWorkerPoolNamesUnique() hcl.Diagnostics {
 
 	return diagnostics
 }
+
+func (c *config) checkCPUManagerPolicy() hcl.Diagnostics {
```

Comment:
Perhaps we should have some tests for this.

Reply:
Added.